### PR TITLE
Remove arcgis, enter END

### PR DESCRIPTION
**Pull request by thorsten-reitz**

### CODE REVIEW

File: src/pages/index.js [hunk 1]
```diff
@@ -72,7 +72,7 @@ export default function Home() {
       title={`Documentation - ${siteConfig.title}`}
       description="hale»connect is the world’s best Software-as-a-Service tool for the agile, data-
       driven design and implementation of more than 5.000 Open Standards such as 
-      INSPIRE, ALKIS, XPlanGML, SOSI, eCH, WFD, FFH, Natura2000, CDDA, and ArcGIS.
+      INSPIRE, ALKIS, XPlanGML, SOSI, eCH, WFD, FFH, Natura2000, CDDA, and END.
```
Comment (stempler) [inline src/pages/index.js:75]: Same text can be found here - not sure what the difference is: https://github.com/wetransform/hc-documentation/blob/f6452abb239628d99584e89ef5a3f74b58f59bef/src/pages/index.js#L31

I think this is where the German version lives: https://github.com/wetransform/hc-documentation/blob/f6452abb239628d99584e89ef5a3f74b58f59bef/i18n/de/code.json#L298

Comment (morch23mj) [reply src/pages/index.js:75]: @thorsten-reitz as @stempler mentioned, [hc-documentation/src/pages/index.js](https://github.com/wetransform/hc-documentation/blob/f6452abb239628d99584e89ef5a3f74b58f59bef/src/pages/index.js#L31) and [hc-documentation/i18n/de/code.json](https://github.com/wetransform/hc-documentation/blob/f6452abb239628d99584e89ef5a3f74b58f59bef/i18n/de/code.json#L298) are the relevant files to update.

The description attribute could be then removed.

**Review by morch23mj — CHANGES_REQUESTED** [review]

File: src/pages/index.js (hunk 1 above)
Comment (thorsten-reitz) [reply src/pages/index.js:75]: Thanks for the pointer, should be fixed now.

**Review by morch23mj — APPROVED** [review]
LGTM

File: i18n/de/code.json [hunk 2]
```diff
@@ -295,7 +295,7 @@
     "message": "populäre Artikel"
   },
   "hero.hcDescription": {
-    "message": "hale»connect ist das weltweit beste Software-as-a-Service-Tool für die agile, datengetriebene Design und Implementierung von mehr als 5.000 offenen Standards wie INSPIRE, ALKIS, XPlanGML, SOSI, eCH, WFD, FFH, Natura2000, CDDA , und ArcGIS. Es reduziert die komplexe Aufgabe der Veröffentlichung von Datensätzen als Dienste, gekoppelt mit Metadaten, auf einen einzigen benutzerfreundlichen automatisierten Workflow."
+    "message": "hale»connect ist das weltweit beste Software-as-a-Service-Tool für die agile, datengetriebene Design und Implementierung von mehr als 5.000 offenen Standards wie INSPIRE, ALKIS, XPlanGML, SOSI, eCH, WFD, FFH, Natura2000, CDDA , und END. Es reduziert die komplexe Aufgabe der Veröffentlichung von Datensätzen als Dienste, gekoppelt mit Metadaten, auf einen einzigen benutzerfreundlichen automatisierten Workflow."
```
Comment (florianesser) [inline i18n/de/code.json:298]: @thorsten-reitz 

"Software-as-a-Service-Tool für die agile, datengetriebene Design und Implementierung" sounds wrong (die Design?)

There's a superfluous space after "CDDA": `Natura2000, CDDA , und END`

**Pull request merged**